sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -100,6 +99,30 @@ class CosmosClient(object):  # pylint: disable=client-accepts-api-version-keywor
     :type credential: Union[str, Dict[str, str], ~azure.core.credentials_async.AsyncTokenCredential]
     :keyword str consistency_level: Consistency level to use for the session. Default value is None (account-level).
         More on consistency levels and possible values: https://aka.ms/cosmos-consistency-levels
+    :keyword int timeout: An absolute timeout in seconds, for the combined HTTP request and response processing.
```

Comment:
We normally link of the the Core documentation for this - as maintaining this list can be difficult.

Reply:
I added this to match what we had in the async client and because I do believe it is information that should be immediately available to users for clarity, specially now that we are planning on rolling out additional documentation on client behavior: https://github.com/Azure/azure-sdk-for-python/pull/28433